<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Efficient RANSAC 2D</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionDetectCurves"/>
    <addaction name="actionDetectLines"/>
    <addaction name="actionRightAngle"/>
    <addaction name="separator"/>
    <addaction name="actionDetectCurvesLines"/>
    <addaction name="separator"/>
    <addaction name="actionGenerateContours"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>Resources/open.png</normaloff>Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDetectContours">
   <property name="text">
    <string>Detect Contours</string>
   </property>
  </action>
  <action name="actionDetectCurves">
   <property name="text">
    <string>Detect Curves</string>
   </property>
  </action>
  <action name="actionDetectLines">
   <property name="text">
    <string>Detect Lines</string>
   </property>
  </action>
  <action name="actionDetectCurvesLines">
   <property name="text">
    <string>Detect Curves/Lines</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>Resources/save.png</normaloff>Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionGenerateContours">
   <property name="icon">
    <iconset>
     <normaloff>Resources/contour.png</normaloff>Resources/contour.png</iconset>
   </property>
   <property name="text">
    <string>Generate Contours</string>
   </property>
  </action>
  <action name="actionRightAngle">
   <property name="text">
    <string>Right Angle</string>
   </property>
  </action>
  <action name="actionDetectCurvesLinesWithRA">
   <property name="text">
    <string>Detect Curves/Lines with RA</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
